infra/aws/terraform/audit-account/securityhub.tf

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+resource "aws_securityhub_organization_admin_account" "default" {
+  admin_account_id = local.audit-account-id
+}
+
+resource "aws_securityhub_organization_configuration" "default" {
+  auto_enable = true
+}
```

Comment:
What region(s) do we want security hub in?

Also see: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/securityhub_finding_aggregator

Reply:
We want all regions to be convered. Some sub-projects of k8s project run tests over all the AWS regions.